configuration/conceptclasses/conceptclasses.csv

```diff
@@ -0,0 +1,3 @@
+Uuid,Void/Retire,Name,Description
```

Comment:
aren't these standard concept names? if yes, then Bahmni should define that out of box. Check with Akhil what config team is doing in this regard

Reply:
@angshu concepts which we are creating here are taken from lite.mybahmni.in environment (CIEL ZIP) and in this project we are using default-config (demo Bahmni). Some concepts are already present in the system but mapping codes are missing. we are updating the mappings also for the existing codes. 

Checked with Akhil, he is going to create comprehensive list of drugs for Bahmni Standard whenever he finds sometime.